builtin/logical/transit/path_encrypt.go

```diff
@@ -154,39 +155,49 @@ func decodeBatchRequestItems(src interface{}, dst *[]BatchRequestItem) error {
 		}
 
 		if v, has := item["context"]; has {
-			if casted, ok := v.(string); ok {
+			val := reflect.ValueOf(v)
```

Comment:
Nit: We may not need `val`. How about `if !reflect.ValueOf(v).IsValid()`?

Reply:
Sounds good! Thank you for the review!